static/src/components/TaskView.vue

```diff
@@ -50,13 +56,13 @@ export default defineComponent({
 </script>
 
 <template>
-  <i-layout-content>
+  <div id="task-view">
     <ProofOfCondition
       v-model:isAttentive="isAttentive"
       v-model:isTrained="isTrained"
     />
     <UserInformationLabel :fullname="fullname() ?? 'Unknown user'" />
-    <div v-if="imageId">
+    <div v-if="imageId && task">
```

Comment:
@jdsauer mentioned [here](https://github.com/team-folivora/annoto/pull/119#discussion_r880149038) that this is somewhat weird, and I think we dont need the check for the `task` object because it gets checked in the nextImage method. But maybe we should implement an earlier check to ensure the `TaskView` is not opened with a non-existing taskId?

Reply:
Okay we ended up doing some fancy error handling and stuff, so this should be safer now. (Dont know if it is better undestandable though)